pulp_2to3_migration/app/plugin/rpm/pulp_2to3_models.py

```diff
@@ -721,19 +720,15 @@ def create_pulp3_content(self):
             self.filename = namespace
             treeinfo_parsed = treeinfo.parsed_sections()
             treeinfo_serialized = TreeinfoData(treeinfo_parsed).to_dict(filename=namespace)
-            # The repository must already be migrated in order to migrate the DistributionTree
-            pulp2_r_c_unit = Pulp2RepoContent.objects.get(pulp2_unit_id=self.pulp2content.pulp2_id)
-            pulp2_repo_id = pulp2_r_c_unit.pulp2_repository.pulp2_repo_id
-            pulp2_repo = Pulp2Repository.objects.get(pulp2_repo_id=pulp2_repo_id, not_in_plan=False)
             # Pulp 2 only knows about the top level kickstart repository
-            treeinfo_serialized["repositories"] = {'.': pulp2_repo.pulp3_repository.pk}
+            treeinfo_serialized["repositories"] = {'.': None}
             # Pulp 2 did not support addon repositories, so we should not list them here either
-            treeinfo_serialized['addons'] = []
+            treeinfo_serialized['addons'] = {}
             # Pulp 2 only supported variants that are in the root of the repository
-            variants = []
-            for variant in treeinfo_serialized['variants']:
+            variants = {}
+            for name, variant in treeinfo_serialized['variants'].items():
                 if variant['repository'] == '.':
-                    variants.append(variant)
+                    variants[name] = variant
```

Comment:
@dkliban , JFYI, this is needed due to your changes when you fixed the CentOS8 kickstart repo publication (the migration of distribution trees )was broken since then)